### ISSUE TITLE
"uninitialized constant Sikulix" on Windows for jruby test

### ISSUE THREAD

**Issue by deeTEEcee** [RaiMan/SikuliX-2014#170]
java version 1.7.0_79
jruby 1.7.22

JAVA_HOME and SIKULIXAPI_JAR are both set to the correct places. I installed pack 1 + pack 2. 

Tested the following code and I get the error that Sikulix is uninitialized.
```
require 'sikulix'
include Sikulix
```

**Comment by RaiMan** [comment]
not really clear, what you are doing.

I suppose we are talking about 1.1.0.

pack 1 means, you have run setup with option 1 and selected only the JRuby option. This allows you to run Ruby scripts from inside the IDE or from command line using runsikulix.cmd -r ...
In both cases `require 'sikulix'; include Sikulix` is not needed, since it is done internally at script run.
No extra JRuby is needed here, since it is bundled with sikulix.jar. 

pack 2 means you have run setup with option 2, which gives you a sikulixapi.jar. To run Ruby scripts, you need an extra JRuby (e.g. version 1.7.22 or the new version 9). A pointer to sikulixapi.jar must be on Java class path at runtime (e.g. with CLASSPATH=<path-to>\sikulixapi.jar) and a script must have  `require 'sikulix'; include Sikulix` before any SikuliX feature is used.

**Comment by rssdev10** [comment]
SIKULIXAPI_JAR must contain filename with path. Not a path only.

I'm using rvm + jruby + running script like:
```bash
#!/usr/bin/sh

SIKULIXAPI_JAR='sikulixapi.jar'
export SIKULIXAPI_JAR

echo $1
ruby $1
```

**Comment by deeTEEcee** [comment]
@RaiMan yes, 1.1.0 sorry about that, i used both packs because i was quite confused what to do. i think im trying to run with pack 2. i originally had only SIKULIXAPI_JAR but i also added the full pathname to the file to CLASS_PATH as well and nothing changes.

switched my jdk to 1.8.0_66-b17 and reinstalled with only pack1 and script seems to work at least in the IDE. ill just go with that then.   

@rssdev10 i do have the full filename with path.

**Comment by rssdev10** [comment]
@deeTEEcee The sikulix.gem from rubygems is obsolete. If you have installed it please replace sikulix-related files directly in installed path by new version https://github.com/RaiMan/SikuliX-2014/tree/master/JRubyGem/lib I guess at nearest time the sikulix.gem will updated.

**Comment by RaiMan** [comment]
I just started with some tests in JRuby and realized, that in the IDE the "auto-require-include" is like this:
require 'Lib/sikulix'
include Sikulix

just try it this way.
